docs/src/pages/@charcoal-ui/react/TextArea/apiData.tsx

```diff
@@ -0,0 +1,72 @@
+import { TextAreaProps } from '@charcoal-ui/react'
+import { ApiTableData } from '../_components/ApiTable'
+
+export const apiData: Omit<
+  ApiTableData<TextAreaProps, HTMLInputElement>,
+  'type' | 'value'
+> = {
+  label: {
+    default: '',
+    description: 'ラベル',
+    required: true,
+    type: 'string',
+  },
+  assistiveText: {
+    description: 'エラーのテキスト',
+    type: 'string',
+  },
+  autoHeight: {
+    default: 'false',
+    description: '高さを自動で変える、multilineのみ有効',
```

Comment:
multiline という属性なくなったからこれは変か

```suggestion
    description: '高さを自動で変える',
```